doc/README.md

```diff
@@ -52,14 +55,17 @@ Some in-development items will have opened issues, as well. Feel free to create
   - Square root
   - Inverse square root
   - Floating point
-    - [Floating-Point Value Types](./components/floating_point.md)
-    - Double (64-bit)
-    - Float (32-bit)
-    - BFloat16 (16-bit)
-    - BFloat8 (8-bit)
-    - BFloat4 (4-bit)
+    - [Floating-Point Value Types](./components/floating_point.md#floatingpointvalue)
+      - Double (64-bit)
+      - Float (32-bit)
+      - BFloat16 (16-bit)
+      - TensorFloat32 (32-bit)
```

Comment:
tf32 is 19 bits?

Reply:
Yes.  I didn't realize and fixed this.  Good catch!